build_tools/benchmarks/trigger_db_import.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Triggers the cloud function which imports the benchmark results from the given
+# GitHub Actions run into the database.
+#
+# Usage: trigger_db_import.sh <importer> <github_run_id> <github_run_attempt>
+#
+# - <importer> is either 'presubmit' or 'postsubmit'
+# - <github_run_id> is an integer identifier
+# - <github_run_attempt> is an integer identifier
+#
+# /<github_run_id>/<github_run_attempt>/ is the path in the bucket where the cloud function is
+# looking for the benchmark results. The bucket is determined by the <importer> option.
+# Note, that you can't trigger an import from an arbitrary bucket for security reasons.
+
+set -euo pipefail
+
+readonly SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+readonly REPO_ROOT="${SCRIPT_DIR}/../../"
+
+source "${REPO_ROOT}/build_tools/github_actions/runner/config/functions.sh"
+
+function usage {
+  echo "Usage: $0 <importer> <github_run_id> <github_run_attempt>" >&2
+  echo "" >&2
+  echo "- <importer> is either 'presubmit' or 'postsubmit'" >&2
+  echo "- <github_run_id> is an integer identifier" >&2
+  echo "- <github_run_attempt> is an integer identifier" >&2
+  exit -1
+}
+
+if [[ $# -ne 3 ]]; then
+  echo -e "Expected 3 arguments, found $#.\n" >&2
+  usage
+fi
+
+if [[ "$1" != "presubmit" && "$1" != "postsubmit" ]]; then
+  echo -e "<importer> can be either 'presubmit' or 'postsubmit'.\n" >&2
+  usage
+fi
+
+if ! [[ "$2" -gt 0 ]] 2>/dev/null; then
+  echo -e "<github_run_id> must be a positive integer!\n" >&2
+  usage
+fi
+
+if ! [[ "$3" -gt 0 ]] 2>/dev/null; then
+  echo -e "<github_run_attempt> must be a positive integer!\n" >&2
+  usage
+fi
+
+readonly WORKFLOW="$1"
+readonly GITHUB_RUN_ID="$2"
+readonly GITHUB_RUN_ATTEMPT="$3"
```

Comment:
Maybe we can move these to the head so the check above can refer to these variables?

Reply:
Yep. Changed.